src/XMakeTasks/UnitTests/ErrorWarningMessage_Tests.cs

```diff
@@ -106,16 +106,17 @@ public void Warning()
         }
 
         /// <summary>
-        /// Empty warning should not log an event
+        /// Empty warning SHOULD log an event
```

Comment:
Looks like it's been this way for all time™️ (since Whidbey). I can't see if it was a late-breaking change then or anything.